wiki/Help_centre/Account_restrictions/vi.md

```diff
@@ -74,48 +68,58 @@ Nếu chúng tôi quyết định cho bạn quay lại, chúng tôi sẽ yêu c
 
-Mong bản hiểu rằng gửi email tới địa chỉ này chỉ đảm bảo bạn được xem xét lại khiếu nại, chứ không đảm bảo thay đổi kết quả gì nếu chúng tôi tìm ra bạn làm sai điều gì trong quá trình khiếu nại.
+Mong bạn hiểu rằng gửi email tới địa chỉ này chỉ đảm bảo bạn được xem xét lại khiếu nại, chứ không đảm bảo thay đổi kết quả nếu chúng tôi tìm ra bạn làm sai điều gì trong quá trình khiếu nại.
 
-**Chúng tôi chỉ phản hồi yêu cầu xem xét lại khiếu nại qua email gửi tới [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh).** Vui lòng không liên lạc tới người thuộc đội ngũ osu! (hoặc người thuộc đội tình nguyện) qua Twitter, Discord, hay bất cứ nền tảng nào khác nhằm lách qua điều khoản này - họ không giúp được bạn đâu.
+**Chúng tôi chỉ phản hồi yêu cầu xem xét/điều tra khiếu nại qua email gửi tới [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh).** Vui lòng không liên lạc tới người thuộc đội ngũ osu! (hoặc người thuộc đội tình nguyện) qua Twitter, Discord, hay bất cứ nền tảng nào khác nhằm lách qua điều khoản này - họ không giúp được bạn đâu.
 
 ## Lý do hạn chế thường gặp và thời gian chờ
 
 | Lý do hạn chế | Thời gian chờ | Chú thích về khiếu nại |
 | :-- | :-- | :-- |
 | Sai sót trong việc đánh giá | Không | Nếu bạn không phạm quy, tài khoản bạn không bị ảnh hưởng gì. |
 | Chơi nhiều tài khoản | Vĩnh viễn (đối với các tài khoản phụ) | Tài khoản chính không bị hạn chế, có thể bị Im lặng. |
-| Chơi nhiều tài khoản (tái diễn) | 3 tháng | Áp dụng các điều về khiếu nại, cấm tham gia giải đấu 1 năm. |
+| Chơi quá nhiều tài khoản | 3 tháng | Áp dụng các điều về khiếu nại, cấm tham gia giải đấu 1 năm. |
 | Chia sẻ tài khoản | 3 tháng | Áp dụng các điều về khiếu nại, cấm tham gia giải đấu 1 năm, rollback một phần. |
-| Gian lận | 3 tháng | Áp dụng các điều về khiếu nại, cấm tham gia giải đấu 1 năm, rollback hoàn toàn. |
-| Có hành vi sai trái quá mức | Thay đổi tuỳ vấn đề | Áp dụng các điều về khiếu nại. Thời gian chờ khiếu nại tuỳ mức độ. |
-| Gian lận/chia sẻ tài khoản trong giải đấu | 6 tháng | Áp dụng các điều về khiếu nại, rollback hợp lý, cấm tham gia giải đấu vĩnh viễn. |
-| Những lần vi phạm tiếp theo | 6 tháng | Áp dụng các điều về khiếu nại, rollback hợp lý, cấm tham gia giải đấu 1 năm với mỗi lần vi phạm. |
+| Những lần vi phạm sau (chơi nhiều tài khoản, chia sẻ tài khoản, hành vi tồi) | 6 tháng | Áp dụng các điều về khiếu nại, rollback (nếu cần), cấm tham gia giải đấu 1 năm nhân số lần phạm quy. |
+| Gian lận | 6 tháng | Áp dụng các điều về khiếu nại, cấm tham gia giải đấu 1 năm, rollback hoàn toàn. |
+| Có hành vi sai trái quá mức | Tuỳ tình huống | Áp dụng các điều về khiếu nại. Thời gian chờ khiếu nại tuỳ mức độ. |
+| Gian lận/chia sẻ tài khoản trong giải đấu | 12 tháng | Áp dụng các điều về khiếu nại, rollback hợp lý, cấm tham gia giải đấu vĩnh viễn. |
+| Những lần "gian lận" tiếp theo | 12 đến 24 tháng | Áp dụng các điều về khiếu nại, rollback, cấm tham gia giải đấu 1 năm nhân số lần vi phạm. |
```

Comment:
```suggestion
| Những lần vi phạm tiếp theo (gian lận) | 12 đến 24 tháng | Áp dụng các điều khoản về khiếu nại, rollback, cấm tham gia giải đấu 1 năm nhân số lần vi phạm. |
```

Reply:
I like to shorten this though, but okay I guess 🤷🏼‍♂️